workflow/src/main/java/com/synopsys/integration/alert/workflow/message/NotificationReceiver.java

```diff
@@ -64,9 +64,11 @@ public void handleEvent(NotificationEvent event) {
                 logger.warn("Can not process a notification event without notification Id's.");
                 return;
             }
+            logger.debug("Event {}", event);
             logger.info("Processing event for {} notifications.", event.getNotificationIds().size());
             List<AlertNotificationModel> notifications = notificationAccessor.findByIds(event.getNotificationIds());
             List<DistributionEvent> distributionEvents = notificationProcessor.processNotifications(FrequencyType.REAL_TIME, notifications);
+            logger.info("Sending {} events for notifications.", distributionEvents.size());
```

Comment:
Do you think this could be too verbose for info? We already mention processing events. I would prefer debug

Reply:
This is very helpful when debugging with the performance issues lately. And the logs serve different purposes. The log on line 68 says how many notifications are being processed and line 71  says how many events were created as a result.